.github/workflows/build-and-tests.yml

```diff
@@ -437,6 +437,11 @@ jobs:
             node: '18.20.0'
             command: 'ci:test:all'
             additionalName: ' with Additional Tests'
+        exclude:
+          # Windows ARM builds were not provided for a while
+          - settings:
+              target: aarch64-pc-windows-msvc
+            node: '18.20.0'
```

Comment:
What do you mean by "not provided for a while"?

Reply:
During Node v12 lifecycle they stopped providing ARM builds because of some "hard to fix bugs". They only manually provided mostly untested unofficial releases from time to time, and only started officially releasing Windows ARM builds for v19+

So there are no Windows ARM releases for v18